backend/site-api/db/sites/migrations/20240118113105_sites_add_content_updated_at.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE
+  site_versions
+ADD COLUMN
+  content_updated_at INTEGER DEFAULT 0;
```

Comment:
I changed it to `INTEGER`, but the original problem is that the migration was written with postgres syntax, which wasn't valid for sqlite3.

It's somewhat annoying to add a date column in sqlite3, you have to add the column with no default, and then fill the data with a separate command since sqlite3 doesn't allow `CURRENT_TIMESTAMP` in `ALTER TABLE` commands.